lib/customisation.nix

```diff
@@ -133,9 +137,7 @@ rec {
       mkAttrOverridable = name: _: makeOverridable (newArgs: (f newArgs).${name}) origArgs;
     in
       if lib.isDerivation pkgs then throw
-        ("function `callPackages` was called on a *single* derivation "
-          + ''"${pkgs.name or "<unknown-name>"}";''
-          + " did you mean to use `callPackage` instead?")
+        ("function `callPackages` was called on a *single* derivation " + ''"${pkgs.name or "<unknown-name>"}";'' + " did you mean to use `callPackage` instead?")
```

Comment:
binops: I'm not sure do we need to wrap it up or not. But looking at binops' rule, there will be no space between `+`

Reply:
The latest commit on nixpkgs-fmt remove auto wrap on binops.